app/views/users/profile-details.njk

```diff
@@ -8,29 +8,12 @@
 
 {% from "macros/page-heading.njk" import pageHeading %}
 
-{% block breadcrumbs %}
-  {{
-    govukBackLink({
-      text: 'Go back to search',
-      href: '/'
-    })
-  }}
-{% endblock %}
-
-{% block content %}
-  {% if error %}
-    {{ govukErrorSummary({
-      titleText: "There is a problem",
-      errorList: error.errorList
-    }) }}
-  {%endif%}
-
-  {% if notification %}
-    {{ govukNotificationBanner(notification) }}
-  {% endif %}
-
-  {{ pageHeading(pageTitle) }}
+{% set backLink = {
```

Comment:
I see there's no presenter for this file and that's probably why you put this here but I think there should be a presenter for this page. the view and the submit have duplicate code for when there is an error and the page is returned. that should all be moved to a presenter and then this can be in the backLink like all our other pages.